### DISCUSSION TITLE
第2节课作业(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#227] [作业]
请提交作业的链接 Please submit a link to the assignment.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/677089410

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/677145690

**Comment by jayjayhust** [comment]
https://zhuanlan.zhihu.com/p/677358431

**Comment by HaroldChen19** [comment]
https://github.com/HaroldChen19/InternLM-Boot-Camp/blob/master/Lesson_2/Assignment.md

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/2-21-1a046f16996e4318bed39a0a26ae5b08

**Comment by harryguantj** [comment]
https://blog.csdn.net/gyzlhg/article/details/135570244

**Comment by XuandYu000** [comment]
https://www.cnblogs.com/viewoverlooking/p/17963189

**Comment by KyleHuang9** [comment]
https://github.com/KyleHuang9/InternLM-tutorial/blob/main/Lesson2/homework.md

**Comment by BoogalooYue** [comment]
Lesson2《轻松玩转书生·浦语大模型趣味Demo——实战部分》
[作业链接](https://www.yuque.com/g/boogalooyue/knowledge/kuaqs992dk53gtxa/collaborator/join?token=5UyQAUtB9IjV8ePr&source=doc_collaborator# )

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/677757604

**Comment by dddxs** [comment]
https://github.com/dddxs/blog/blob/main/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B-%E7%AC%AC%E4%BA%8C%E6%AC%A1%E4%BD%9C%E4%B8%9A.md

**Comment by YongXie66** [comment]
https://zhuanlan.zhihu.com/p/677792678

**Comment by yuxudong20** [comment]
https://github.com/InternLM/tutorial/discussions/143#discussioncomment-8035693

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/677722729
作业连接（与笔记连接是同一个）

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/2
